source/process/community-overview.rst

```diff
@@ -0,0 +1,74 @@
+============================================================
+Mattermost Community
+============================================================
+
+.. contents::
+  :backlinks: top
+  :local:
+  :depth: 2
+
+----
+
+Vision
+---------------------------------------------------------
+
+Increase popularity of Mattermost by empowering the community to have an impact by contributing features and extensions that would not otherwise be offered
+
+Principles
+---------------------------------------------------------
+
+1. Focus on areas with high impact-to-mana ratio.
+2. It should be easy and straightforward to start contributing.
+3. Community building should be directed towards 3R’s: Recognition, Retention and Recruitment
+
+Reality Check
+---------------------------------------------------------
+
+1. You cannot attract contributors without attracting users.
+  
+  - For every 1000 users, 100 file bugs, 10 submit a patch, 1 reads the contribution guidelines
+
+2. Set realistic expectations to contributors.
+
+  - Majority of contributors will not be experts. Also, contributing to open source is typically a hobby / side job.
+
+3. There's thousands of other open source projects so it's very easy to leave to another project.
+
+4. Community building is mana intensive.
+
+  - You need to train contributors, dev and PM review PRs, and create recognition programs with specific roles to people.
```

Comment:
Propose replacing with:
"You need to train contributors, developers and PMs to review PRs and create recognition programs that give specific roles to people."